modules/ROOT/pages/design-common-problems-raml-10.adoc

```diff
@@ -811,7 +811,7 @@ title: ExampleRAML
 ----
 {
 ...
-"assesments.characteristic.focusofattention.data"= "",
+"assessments.characteristic.focusofattention.data"= "",
```

Comment:
this looks like a typo, but maybe it isn't? I updated it but please note if it is supposed to be spelled that way so I can revert.